.rubocop-https---raw-githubusercontent-com-standout-Coding-master-ruby-rubocop-yml

```diff
@@ -0,0 +1,40 @@
+AllCops:
```

Comment:
At first I thought so too, but it could be usable in two scenarios : 

A) offline development 
B) tracking changes in the file

But I'm not totally convinced.